packages/dashboard/pages/api/1.0/index.ts

```diff
@@ -0,0 +1,8 @@
+import withService from '../../../middleware/withService';
+import { uiConfigs } from '../../../config';
+
+const handler = async (req: any, res: any) => {
+  res.json(uiConfigs);
```

Comment:
we don't need an endpoint to expose UI configs.

Reply:
Updated PR!